charts/plgd-hub/values.yaml

```diff
@@ -432,6 +432,8 @@ resourcedirectory:
     mountPath: /config
   # -- Service and POD port
   port: 9100
+  # -- Hub ID. Can be overriden via global.hubId
```

Comment:
Can be overriden or this can override the global.hubId ?

Reply:
fixed